packages/front-end/components/Experiment/UrlRedirectModal.tsx

```diff
@@ -231,26 +232,39 @@ const UrlRedirectModal: FC<{
         <div className="mt-3">
           <h4>Destination URLs</h4>
           {experiment.variations.map((v, i) => {
-            let warning: string | undefined;
+            let warning: string | JSX.Element | undefined;
             const destinationMatchesOrigin =
               !!form.watch("originUrl") &&
               form.watch(`destinationUrls.${i}`) &&
-              (isURLTargeted(form.watch("originUrl"), [
+              (isURLTargeted(form.watch(`destinationUrls.${i}`), [
                 {
                   include: true,
                   type: "simple",
-                  pattern: form.watch(`destinationUrls.${i}`),
+                  pattern: form.watch("originUrl"),
                 },
               ]) ||
                 form.watch("originUrl") === form.watch(`destinationUrls.${i}`));
             try {
               const originUrl = new URL(form.watch("originUrl"));
               const variantUrl = new URL(form.watch(`destinationUrls.${i}`));
               if (originUrl.protocol !== variantUrl.protocol) {
-                warning = `Destination URL is using "${variantUrl.protocol}" and the origin URL is using "${originUrl.protocol}"`;
+                warning = `Destination URL is using "${variantUrl.protocol}" and the original URL is using "${originUrl.protocol}"`;
               }
               if (originUrl.host !== variantUrl.host) {
-                warning = `Destination URL is "${variantUrl.host}" and the origin URL is "${originUrl.host}"`;
+                warning = `Destination URL is "${variantUrl.host}" and the original URL is "${originUrl.host}"`;
+              }
+              if (
+                !originUrl.searchParams.size &&
+                variantUrl.searchParams.size
+              ) {
+                warning = (
+                  <>
+                    Destination URL has query parameters but the original URL
+                    does not. See{" "}
+                    <DocLink docSection="url_redirects">our docs</DocLink> for
```

Comment:
consider linking to the #subsection of the document. maybe not necessary if the doc itself is small though...